dist/get.kobman.io

```diff
@@ -187,7 +184,7 @@ echo "kobman_colour_enable=true" >> "$kobman_config_file"
 echo "Download script archive..."
 
 # once move to kobman namespace needs to update kobman-latest.zip 
-curl --location --progress-bar "${KOBMAN_SERVICE}/${KOBMAN_NAMESPACE}/KOBman/master/dist/kobman-latest.zip" > "$kobman_zip_file"
+curl --location --progress-bar "${KOBMAN_SERVICE}/${KOBMAN_NAMESPACE}/KOBman/${KOBMAN_DIST_BRANCH}/dist/kobman-latest.zip" > "$kobman_zip_file"
```

Comment:
should be ${KOBMAN_SERVICE}/kobman-latest.zip based on the change in line 8

Reply:
This has been modified as instructed , but the download is not happening as KOBMAN_SERVICE is not getting populated with the expected value.